File: c/3-loops/practice/lp-4.c
#include <stdio.h>
int main()
{
    int n ,odd =1;
    scanf("%d",&n);
    if(n<0)
    {
        printf("Invalid input");
        return 0;
    }
    for(int i=1;i<=n;i++)
    {
        for(int s=0;s<n-i;s++)
        {
            printf(" ");
        }
        for(int j =0;j<i;j++)
        {
            printf("%d ",odd);
            odd+=2;
        }
        printf("\n");
    }
    return 0;
}

File: python/7-lists/practice/lp-4.py
stock = list(map(int, input().split()))

remove = list(map(int, input().split()))


if any(x < 0 for x in stock) or any(x < 0 for x in remove):
    print("Invalid input")
else:

    remaining = [item for item in stock if item not in remove]
    print(*remaining)